tool/src/main/java/org/wildfly/security/tool/VaultCommand.java

```diff
@@ -445,7 +445,7 @@ private void printSummary (String keystorePassword, String salt, int iterationCo
         if (keystorePassword != null) {
             password = keystorePassword;
             if (salt != null && iterationCount > -1) {
-                password = keystorePassword.startsWith("MASK-") ? keystorePassword + ";" + salt + ";" + String.valueOf(iterationCount)
+                password = keystorePassword.startsWith("MASK-") ? keystorePassword + ";" + salt + ";" + iterationCount
```

Comment:
Also, as an inside, I was wondering if it would be good to add [dev containers](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers) support in the project. In my understanding, it would make it trivial for any developer to get the environment set up for the project

Reply:
That's an interesting suggestion. We'll look into that some more to see if it makes sense.